examples/templates/Example.js

```diff
@@ -206,6 +217,32 @@
             if ( optionalArgs )
                 osg.extend( optionsDebug, optionalArgs );
 
+            if ( !this._programRTT ) {
```

Comment:
It's because your texture are 0 so textureNull, so not used in the compiler to build the program. I am not against having a specific program for that, but I dont want you workaround stuff if you dont understand why it does not work.